presto-docs/src/main/sphinx/release/release-0.239.rst

```diff
@@ -0,0 +1,79 @@
+
+General Changes
+_______________
+* Fix :func:`classification_miss_rate` and :func:`classification_fall_out` (:pr:`14740`).
+* Fix ``NullPointerException`` in ``/v1/thread`` end point.
+* Fix an issue where the property ``ignore_stats_calculator_failures`` would not be honored
+  for certain queries that had filters pushed down to the table scan.
+* Fix missing query completion events for queries which fail prior to dispatching.
+* Fix potential infinite loop when the setting ``use_legacy_scheduler`` is set to ``false``.
+* Optimize queries with repeated expressions in filters or projections by computing the
+  common expressions only once. This can be disabled by the session property
+  ``optimize_common_sub_expressions``.
+* Optimize queries containing only :func:`min` and :func:`max` on columns that can be
+  evaluated using metadata (e.g., Hive partitions). This is controlled by configuration property
+  ``optimizer.optimize-metadata-queries`` and session property ``optimize_metadata_queries``.
+  Note: Enabling this optimization might change query result if there are metadata that refers to
+  empty data, see :pr:`14845` for examples.
+* Add aggregation function :func:`set_union`.
+* Add local disk spilling support for queries with ``ORDER BY`` or ``DISTINCT``.
+* Add new unified grouped execution configuration property ``grouped-execution-enabled`` and
+  session property ``grouped_execution`` with default set to ``true``. The property
+  ``grouped-execution-for-join-enabled`` will be removed in a future release (:pr:`14886`).
+* Remove experimental feature to perform grouped execution for eligible table scans and its
+  associated configuration property ``experimental.grouped-execution-for-eligible-table-scans``
+  and session property ``grouped_execution_for_eligible_table_scans``.
+* Enable ``dynamic-schedule-for-grouped-execution`` by default.  This property will be removed
+  in a future release.
+* Enable ``grouped-execution-for-aggregation-enabled`` by default. This property will be removed in
+  a future release.
+* Enable async page transport with non-blocking IO for exchange by default. This can be disabled by
+  setting the configuration property ``exchange.async-page-transport-enabled`` to ``false``.
+
+
+Security Changes
+----------------
+* Add support for ``DEFINER`` and ``INVOKER`` view security modes. While querying a view the former
+  uses the permissions of the view owner and the latter uses the query runner's permissions.
+  See :doc:`/sql/create-view`.
+
+JDBC Changes
+____________
+* Implemented ``DatabaseMetaData#getClientInfoProperties`` API.
+
+Web UI Changes
+--------------
+*  Fix a bug where the UI for the worker's thread snapshot wouldn't display the stack trace.
+
+Hive Changes
+____________
+* Add support for caching the Glue metastore.
+* Add support for warning on unfiltered partition keys. This can be enabled using the configuration
+  property ``partition-keys-to-warn-on-no-filtering``.
```

Comment:
What is the default setting?

Reply:
Generally when we say enable using, my assumption is that it is disabled by default.